TWTValidation/Validators/Value Validators/TWTStringValidator.h

```diff
@@ -76,6 +76,18 @@
 + (TWTBoundedLengthStringValidator *)stringValidatorWithMinimumLength:(NSUInteger)minimumLength maximumLength:(NSUInteger)maximumLength;
 
 /*!
+ @abstract Creates and returns a new string validator that validates that strings are of the specified strings
+ have the specified minimum and maximum lengths, given by number of composed characters.
+ @discussion This method creates a new TWTBoundedComposedCharacterLengthStringValidator with the appropriate minimum and
+ maximum lengths.
```

Comment:
Indent by four characters. Do this for all the documentation that starts with an @…